### PR TITLE
Updating the Readme File of SUI Studio

### PR DESCRIPTION
**Pull request by turolopezsanabria**
Proposing content for the homepage of SUI Studio

adding info to appear here

### CODE REVIEW

**Review by jcger — APPROVED** [review]

**Review by adriadoz — APPROVED** [review]

File: components/README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+- Improve and existing component:
+
+## Adevinta's Design System · Resources
+
+SUI is a fundamental part of Adevinta's Design System, wich also includes UI Kits, Tools and Processes to facilitate how we collaborate and share knowledge across teams and projects, bringing support for Designers, Product Managers and Developers.
+
+- Download the Sketch UI-Kit:
+- SUI tools: 
+- Get in touch: Design-Ops, Development:
+- Status Dashboard: https://pages.github.mpi-internal.com/scmspain/sui-dashboard/
```
Comment (midudev) [inline components/README.md]: I think this URL might not be accesible from external office IPs

Comment (turolopezsanabria) [reply components/README.md]: yeap... I'm aware, I still think it's a good link to have there, even if it's a dead-end for external users. What do you think?

Comment (jcger) [reply components/README.md]: Maybe you could add a warning or something

Comment (turolopezsanabria) [reply components/README.md]: Sure, that's a good idea
Changes done, please check now

**Review by midudev — APPROVED** [review]

**Pull request merged**